examples/product-documents/ecs/usage-examples/python/test/test_demo.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import unittest
+import ros_cdk_core as core
+from demo.demo_stack import DemoStack
+
+
+class TestStack(unittest.TestCase):
```

Comment:
Better to fill in the unit tests. If you don't plan on unit testing, you can delete the test folder.

Reply:
Removed test/ folder